### PR TITLE
reset store information start_ts/uptime when tikv is down

### PR DESCRIPTION
**Pull request by jomenxiao**
#634

### CODE REVIEW

**Comment by siddontang** [comment]
what is the new output now?

**Comment by disksing** [comment]
@jomenxiao Please sign CLA. The changes LGTM, but I don't think it fixes the issue, have you test it out?

**Comment by jomenxiao** [comment]
```
» store 15
{
  "store": {
    "id": 15,
    "address": "172.16.10.14:30163",
    "state": 0,
    "state_name": "Down"
  },
  "status": {
    "store_id": 15,
    "capacity": "246 GiB",
    "available": "214 GiB",
    "leader_count": 0,
    "region_count": 5,
    "sending_snap_count": 0,
    "receiving_snap_count": 0,
    "applying_snap_count": 0,
    "is_busy": false,
    "start_ts": "1970-01-01T08:00:00+08:00",
    "last_heartbeat_ts": "2017-08-10T21:06:07.999684725+08:00",
    "uptime": "0s"
  }
}
```
@siddontang  @disksing 
just reset start_ts/uptime

File: server/store.go [hunk 1]
```diff
@@ -187,13 +187,16 @@ func (s *StoreStatus) clone() *StoreStatus {
...
+	}
 	return time.Unix(int64(s.GetStartTime()), 0)
 }
 
 // GetUptime returns the uptime.
 func (s *StoreStatus) GetUptime() time.Duration {
 	uptime := s.LastHeartbeatTS.Sub(s.GetStartTS())
-	if uptime > 0 {
+	if uptime > 0 && !s.IsDown() {
 		return uptime
```
Comment (siddontang) [inline server/store.go]: any test to cover it?

File: server/store.go [hunk 2]
```diff
@@ -187,13 +187,16 @@ func (s *StoreStatus) clone() *StoreStatus {
 
 // GetStartTS returns the start timestamp.
 func (s *StoreStatus) GetStartTS() time.Time {
+	if s.IsDown() {
+		return time.Unix(0, 0)
```
Comment (siddontang) [inline server/store.go]: any test?

**Comment by siddontang** [comment]
@disksing 

Is it the output proper now?

**Comment by siddontang** [comment]
any update? 

@jomenxiao @disksing

**Comment by disksing** [comment]
We need to think of another approach, close it for now.

**Pull request closed without merging**